transforms/lmru_painpoints.py

```diff
@@ -0,0 +1,21 @@
+{
+    'sheet': 'Pain Bright',
+    'lets': {
+        'iri': (
+            'vm:_painpoint'
+            '_{row[Stakeholder].as_slug}'
+            '_{row[Painpoint / Bright spot title].as_slug}'),
+        'stakeholder': 'vm:_stakeholder_{row[Stakeholder].as_slug}',
+    },
+    'non_unique': ['vm:name', 'vm:description'],
+    'triples': [
+        ('{iri}', 'rdf:type', 'vm:Painpoint'),
+        ('{iri}', 'vm:harmsCapitalType', '{row[Capital type].as_capital}'),
+        ('{iri}', 'vm:name', '{row[Painpoint / Bright spot title].as_text}'),
+        ('{iri}', 'vm:description',
+            '{row[Painpoint / Bright spot description].as_text}'),
+        ('{iri}', 'vm:ofStakeholder', '{stakeholder}'),
```

Comment:
Slight gotcha to be aware of when using `lets` values in the `obj` field that I found a couple of weeks back -- usually if `stakeholder` was missing you'd want the entire triple to be dropped, but this won't happen if the variable is being interpolated into a string in `lets` as you get the stub value instead.